sql/rpl_parallel.cc

```diff
@@ -2435,21 +2435,9 @@ rpl_parallel::stop_during_until()
 bool
 rpl_parallel::workers_idle()
 {
-  struct rpl_parallel_entry *e;
```

Comment:
What do you think to keep this block in `#ifndef DBUG_OFF` and assert its `return` is equal the main branch's one?